python/ray/autoscaler/docker.py

```diff
@@ -46,20 +47,26 @@ def with_docker_exec(cmds,
     ]
 
 
-def check_docker_running_cmd(cname):
+def _check_helper(cname, template):
     return " ".join([
-        "docker", "inspect", "-f", "'{{.State.Running}}'", cname, "||", "true"
+        "docker", "inspect", "-f", f"'{{{{{template}}}}}'", cname, "||", "true"
```

Comment:
This is a bit hard to read - I wouldn't recommend this otherwise but can you do:

```
"'{{" + template + "}}'"

Reply:
Sounds good--I wasn't sure if this was too weird (5 brackets is really quite verbose)